product-service/serverless.ts

```diff
@@ -39,7 +49,32 @@ const serverlessConfiguration: AWS = {
       PG_DATABASE,
       PG_USERNAME,
       PG_PASSWORD,
+
+      REGION,
+
+      PRODUCTS_SUBSCRIPTION_EMAIL,
+      EXPENSIVE_PRODUCTS_SUBSCRIPTION_EMAIL,
+
+      CREATE_PRODUCT_TOPIC_ARN: {
+        Ref: "CreateProductTopic",
+      },
     },
+    iamRoleStatements: [
+      {
+        Effect: "Allow",
```

Comment:
unnecessary permission, it added after setting sqs as a trigger in function config